packages/flutter/lib/src/material/segmented_button.dart

```diff
@@ -190,6 +191,16 @@ class SegmentedButton<T> extends StatefulWidget {
   /// [onSelectionChanged] will not be called.
   final bool emptySelectionAllowed;
 
+  /// Determines whether the segmented button should expand to fill
+  /// the available space.
+  /// If true, the segmented button expands to fill its parent's
+  /// horizontal space, evenly dividing this space among its segments.
```

Comment:
nit: The first paragraph of a doc should be a one-sentence summary, see https://dart.dev/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary

Just add a blank line after line 195.